src/_guides/language/effective-dart/usage.md

```diff
@@ -101,7 +101,6 @@ my_package
 
 And say `api_test.dart` imports `api.dart` in two ways:
 
-{:.bad}
```

Comment:
We should leave this line in. It's **bad** to import the same library two different ways.